Update 2021 summit agenda

magnusbaeck opened the pull request:
### Applicable Issues
N/A

### Description of the Change
As discussed during the agenda planning we should have comprehensive abstracts/summaries for everything on the 2021 summit agenda. Also added an owner to all items and replaced the 15:00 session on the first day. All changes were discussed at the 2021-09-02 planning meeting.

### Alternate Designs
None.

### Benefits
Better summit agenda.

### Possible Drawbacks
None.

### Sign-off
<!-- Sign the below certificate of origin, using your full name and e-mail address. -->
<!-- The certificate is copied from https://developercertificate.org/ -->

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I    have the right to submit it under the open source license    indicated in the file; or

(b) The contribution is based upon previous work that, to the best    of my knowledge, is covered under an appropriate open source    license and I have the right under that license to submit that    work with modifications, whether created in whole or in part    by me, under the same open source license (unless I am    permitted to submit under a different license), as indicated    in the file; or

(c) The contribution was provided directly to me by some other    person who certified (a), (b) or (c) and I have not modified    it.

(d) I understand and agree that this project and the contribution    are public and that a record of the contribution (including all    personal information I submit with it, including my sign-off) is    maintained indefinitely and may be redistributed consistent with    this project or the open source license(s) involved.

Signed-off-by: Magnus Bäck \<magnus.back@axis.com\>

magnusbaeck commented:
@e-backmark-ericsson, @t-persson, @m-linner-ericsson, please have a look.

One thing I thought about but I didn't want to deal with right now: Should there be one or more organizer contact addresses for questions about the summit? Apart from general questions, if we ask people to tell us when they have something to share in the "What's cooking" sessions there should be a one or more contact addresses (we could of course just include our own addresses).

e-backmark-ericsson commented:
We could use the just now created google group as contact address - eiffel-tc@googlegroups.com

magnusbaeck commented:
> We could use the just now created google group as contact address - eiffel-tc@googlegroups.com

Yeah, good idea. I've added the address to the bullet list at the top and added a paragraph that encourages people to let us know about their short "what's cooking" talks.
